src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -102,6 +102,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
     <DebugSymbols Condition="'$(DebugSymbols)'==''">false</DebugSymbols>
     <CheckForOverflowUnderflow Condition="'$(CheckForOverflowUnderflow)'==''">false</CheckForOverflowUnderflow>
+
+    <WarningsAsErrors>NU1605</WarningsAsErrors>
```

Comment:
It is disconcerting to me that we're going to be passing NU1605 to the compiler as well as honoring it in nuget. 

I had a look and Roslyn will ignore these identifiers, but fsc will not. This is going to break F#:
```
FSC : error FS0241: 'NU1605' is not a valid integer argument [D:\Temp\wae\fsc\fsc.fsproj]
```

@srivatsn, @rrelyea, I don't think we should be passing the same properties to the compiler and nuget.